### PR TITLE
Added additional instructions for Windows

### PR DESCRIPTION
**Pull request by asheroto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,14 @@ This environment includes the latest versions of the following software:
...
  - Docker (https://www.docker.com/products/docker-desktop)
  - Tested on macOS 10.15 (Catalina) and Windows 10 Version 10.0.18362
- - Windows 10 requires Docker File Sharing with the C drive to be enabled in the Docker settings, restart may be required after enabling sharing
+
+### Windows 10 Prerequisites
+
+- In Docker settings, file sharing must be enabled for the **C:\\** folder
+	- Located under the **Resources** section, then the **File Sharing** section as seen in [this image](https://i.imgur.com/maub2Wh.png).
+- If you do not see this option, make sure you are using **Windows containers**
+	- Right-click the Docker icon in the system tray and click **Switch to Windows containers** as seen in [this image](https://i.imgur.com/enYmIPH.png)
```
Comment (jlucki) [inline README.md:22]: Hey @asheroto, thanks for the contribution! More detailed instructions is always a good thing. One question, I'm running Linux containers just fine on Windows 10, were you not able to enable File Sharing without switching to Windows containers? Which version of Docker are you on?

**Comment by asheroto** [comment]
Correct.  I was using Linux containers but had to switch to Windows containers and disable WSL2 integration in order for File Sharing to show up.  I'm using the latest... 

image

**Comment by jlucki** [comment]
> Correct. I was using Linux containers but had to switch to Windows containers and disable WSL2 integration in order for File Sharing to show up. I'm using the latest...

Ok, I just want to make sure we're giving the correct instructions. Linux containers work just as well on Windows 10. Looking here: https://docs.docker.com/desktop/windows/ this is key:

- The File sharing tab is only available in Hyper-V mode, because in WSL 2 mode and Windows container mode all files are automatically shared by Windows.

Sounds like if you're running windows containers you don't need to configure the File Sharing, which is why it's hidden.

How about we modify that section to say something like:

If you're still having trouble running the containers and starting your app, try switching to Windows containers. To do this... and then your last set of instructions.

**Comment by asheroto** [comment]
Ah, that works.  I'll close this PR.

**Pull request closed without merging**